docs/reference/rest-api/logs.asciidoc

```diff
@@ -0,0 +1,278 @@
+[role="xpack"]
+[[logs-api]]
+== Log ingestion API
+
+experimental::[]
+
+Provides a simple JSON API to ingest log events into {es}.
+
+[discrete]
+[[logs-api-request]]
+=== {api-request-title}
+
+`POST /_logs`
+
+`POST /_logs/<dataset>`
+
+`POST /_logs/<dataset>/<namespace>`
+
+[discrete]
+[[logs-api-prereqs]]
+=== {api-prereq-title}
+* If the {es} {security-features} are enabled, you must have the `create`
+<<privileges-list-indices,index privileges>> for the target data stream.
+* As the target stream depends on routing rules and the data provided in the log events,
+it's recommended to grant permissions for `logs-*-*`.
+* Automatic data stream creation requires a matching index template with data
```

Comment:
As the `logs-*-*` template already exist, I assume a matching template always exists either the global one or one created by users?